app/src/views/private/components/flow-dialogs.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { FlowDialogsContext, injectRunManualFlow } from '@/composables/use-flows';
+import { useI18n } from 'vue-i18n';
+
+const { t } = useI18n();
+
+defineProps<FlowDialogsContext>();
+
+const { runManualFlow } = injectRunManualFlow();
+
+function handleRunManualFlow(flowId: string, isActionDisabled = false) {
+	runManualFlow(flowId, isActionDisabled);
+}
+</script>
+
+<template>
+	<v-dialog
+		:model-value="displayUnsavedChangesDialog.value"
+		keep-behind
+		@esc="resetConfirm"
+		@apply="confirmUnsavedChanges(confirmRunFlow.value!)"
+	>
+		<v-card>
+			<v-card-title>{{ t('unsaved_changes') }}</v-card-title>
+			<v-card-text>{{ t('run_flow_on_current_edited_confirm') }}</v-card-text>
+
+			<v-card-actions>
+				<v-button secondary @click="resetConfirm">
+					{{ t('cancel') }}
+				</v-button>
+				<v-button @click="confirmUnsavedChanges(confirmRunFlow.value!)">
+					{{ confirmButtonCTA }}
+				</v-button>
```

Comment:
@formfcw Bug that was fixed was that when a field required confirmation AND there were unsaved changes, the button would be permanently disabled.

I removed the disabled state as this button should never be disabled if the dialog is present.